webrtc/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>webrtc</artifactId>
+	<version>0.0.1</version>
+
+	<parent>
+		<artifactId>parent-boot-1</artifactId>
+		<groupId>com.baeldung</groupId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../parent-boot-1</relativePath>
```

Comment:
Is there a particular reason why you chose Spring Boot 1.x? We have a parent-boot-2 module for 2.x apps.

Reply:
No, I have now updated the pull request to use parent-boot-2.